docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -357,6 +358,10 @@ We have now configured an alert rule and a contact point. Now let's see if we ca
 
 Once the query `sum(rate(tns_request_duration_seconds_count[5m])) by(route)` returns a value greater than `0.2` Grafana will trigger our alert. Browse to the Request Bin we created earlier and find the sent Grafana alert notification with details and metadata.
 
+{{< admonition type="note"
```

Comment:
```suggestion
{{< admonition type="note" >}}
```
Could be this?

Reply:
Ah that was all me!! That's probably it!